### DISCUSSION TITLE
Does Bundling In CDK Libraries Allow End Users To Consume Bundle?

### DISCUSSION THREAD

**Discussion by bestickley** [projen/projen#3628] [Q&A] [answered]
Does [Bundling](https://projen.io/docs/concepts/bundling) support CDK Libraries so that consumers of the library don't have to bundle the code themselves? It's seems like it does not because bundled code is written to `assets/` and only `lib/` is deployed to NPM, right?

**Comment by mrgrain** [comment, marked as answer]
Files in `assets/` are by default included in the npm package.

Here is what happens on my machine:

```console
$ projen new awscdk-construct
$ echo "hello" > assets/test
$ echo "hello" > assets/test.js
$ ls assets/
total 8
-rw-r--r--@  6B  4 Jun 11:46 test
-rw-r--r--@  6B  4 Jun 11:49 test.js
$ projen build
$ tar xvzf dist/js/pj-bundle@0.0.0.jsii.tgz
x package/.jsii
x package/LICENSE
x package/assets/test
x package/lib/index.js
x package/assets/test.js
x package/package.json
x package/API.md
x package/README.md
x package/lib/index.d.ts
```
